frontend/src/components/App/App.css

```diff
@@ -147,7 +147,7 @@ a {
 }
 
 .Button.-Primary path {
-  fill: var(--primary) !important;
```

Comment:
wait what, this is button primary, plz don't update color of this. make a button secondary and use that